<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTetris</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resurses.qrc">
    <normaloff>:/Images/Images/main.svg</normaloff>:/Images/Images/main.svg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Images/Images/back.jpg);
color: #4B7FFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	background: rgba(255, 255, 255, 0.0);;
	color:green;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
    <item row="0" column="1" rowspan="2">
     <widget class="SceneView" name="sceneView" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>400</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">	background: rgba(255, 255, 255, 0.5);
	color:green;</string>
      </property>
      <property name="rows" stdset="0">
       <UInt>30</UInt>
      </property>
      <property name="columns" stdset="0">
       <UInt>30</UInt>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="scoreStr">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>SCORE: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="scoreNum">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="nextStr">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>next figure</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>NEXT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Lab1_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>nest figure</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="speedStr">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="toolTip">
         <string>game speed</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#speedStr{
color: #BB1D1D;
}</string>
        </property>
        <property name="text">
         <string>SPEED</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="speedNum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>game speed</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit#speedNum{
color:#BB1D1D;
}</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>&amp;Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionMode"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuSettings"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(46, 38, 38, 0);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Game"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="actionMode"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionNew_Game">
   <property name="icon">
    <iconset resource="Resurses.qrc">
     <normaloff>:/Images/Images/new.svg</normaloff>:/Images/Images/new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="Resurses.qrc">
     <normaloff>:/Images/Images/load.ico</normaloff>:/Images/Images/load.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resurses.qrc">
     <normaloff>:/Images/Images/save.ico</normaloff>:/Images/Images/save.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resurses.qrc">
     <normaloff>:/Images/Images/exit.svg</normaloff>:/Images/Images/exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Resurses.qrc">
     <normaloff>:/Images/Images/info.ico</normaloff>:/Images/Images/info.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionMode">
   <property name="icon">
    <iconset resource="Resurses.qrc">
     <normaloff>:/Images/Images/mode.svg</normaloff>:/Images/Images/mode.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Mode</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneView</class>
   <extends>QWidget</extends>
   <header>sceneview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resurses.qrc"/>
 </resources>
 <connections/>
</ui>
